test/reductions/test_grid_search_classification.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from fairlearn.metrics import DemographicParity
+from fairlearn.reductions import GridSearchClassification
+
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+
+
+class TestGridSearchClassification:
```

Comment:
just generally: pytest doesn't need these classes, so you can save yourself from indenting everything

Reply:
It's something of a habit for me. Looking at how Python  deals with files-as-modules, I should probably remove them.